### DISCUSSION TITLE
I use windows (DNS) but it does not block adult sites.

### DISCUSSION THREAD

**Discussion by Corineus** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#697] [Q&A]
Is this a problem due to https or does the dns server not block adult sites?

**Comment by spirillen** [comment]
DNS servers have no influence on the protocol you are using for net-traffic. 

Weather its might be clocking Adult site is somewhat easy to test, if it are blocking for domains like pornhub.com, well I suspect it is block porn domains...
